sys/net/gnrc/link_layer/lorawan/include/gnrc_lorawan_internal.h

```diff
@@ -254,6 +254,17 @@ void gnrc_lorawan_generate_session_keys(const uint8_t *app_nonce,
  */
 int gnrc_lorawan_set_dr(gnrc_lorawan_t *mac, uint8_t datarate);
 
+/**
+ * @brief Set TX power (PHY parameter) for the next transmission
+ *
+ * @param[in] mac pointer to the MAC descriptor
+ * @param[in] tx_pwr desired TX power index
+ *
+ * @return 0 on success
+ * @return -EINVAL if TX power is not available in the current region
```

Comment:
You should add another return with "@return negative errno". Since there's a call to `dev->driver->set` there could be other return codes as well

Reply:
Done